assets/javascripts/modules/analytics/cmp.js

```diff
@@ -34,4 +37,12 @@ const checkCCPA = () => new Promise((resolve) => {
 
 const registerCallbackOnConsentChange = (fn) => onConsentChange(fn);
 
-export { getConsentForVendors, checkAllTCFv2PurposesAreOptedIn, checkCCPA, registerCallbackOnConsentChange };
+const createPrivacySettingsLink = () => {
```

Comment:
I'm not sure `createPrivacySettingsLink` should remain in the `analytics.js` module now we're calling it from `main.js`, I think it makes sense to move the entire function into `main.js`. `analytics.js` is primarily concerned with making tracking-analytics calls.

Reply:
Makes sense, we have the CMP module in scope there too. I'll make the change.